docs/knowledgebase/getting-started/windows-users.md

```diff
@@ -43,34 +43,45 @@ If you do decide to try and use a Windows computer to _natively_ build Substrate
    rustup target add wasm32-unknown-unknown --toolchain nightly
    ```
 
-4. Install LLVM: https://releases.llvm.org/download.html
-
-5. Install OpenSSL with `vcpkg` using PowerShell:
-
-   ```bash
-   mkdir C:\Tools
-   cd C:\Tools
-   git clone https://github.com/Microsoft/vcpkg.git --depth=1
-   cd vcpkg
-   .\bootstrap-vcpkg.bat
-   .\vcpkg.exe install openssl:x64-windows-static
-   ```
-
-6. Add OpenSSL to your System Variables using PowerShell:
-
-   ```powershell
-   $env:OPENSSL_DIR = 'C:\Tools\vcpkg\installed\x64-windows-static'
-   $env:OPENSSL_STATIC = 'Yes'
-   [System.Environment]::SetEnvironmentVariable('OPENSSL_DIR', $env:OPENSSL_DIR, [System.EnvironmentVariableTarget]::User)
-   [System.Environment]::SetEnvironmentVariable('OPENSSL_STATIC', $env:OPENSSL_STATIC, [System.EnvironmentVariableTarget]::User)
-   ```
-
-7. Install `cmake`: https://cmake.org/download/
-
-8. Install `make`
-    - This can be done using Chocolatey. First you need to install the Chocolatey package manager: https://chocolatey.org/install
-    - Once Chocolatey installed you can install make:
-
-   ```
-   choco install make
-   ```
+4. Install dependencies:
+   - A simple way is using **Chocolatey**. 
+      - First, install the [Chocolatey package manager](https://community.chocolatey.org/courses/installation/installing)
+      - Then, you can install LLVM, OpenSSL, cmake and make from either cmd or PowerShell:
+         ```powershell
+         choco install llvm openssl cmake make -y
```

Comment:
https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/1083
About LLVM, I'm not testing the use of chocolatey to install LLVM have this question. does it work? because I'm used scoop install LLVM does not work.

Reply:
chocolatey works in my test. how's your testing going?

Reply:
I have run into "linker `link.exe` not found" after installing dependencies with chocolatey, so is safe to say that it is worth it set up LLVM by hand.

Reply:
> "linker `link.exe` not found"
this looks like an error about **vs_buildtools** installation. Did you install that?

Reply:
sorry, I'm not testing use choco, but I read the code.

https://github.com/bspeice/au-packages/blob/e8d7a320a90c812c7e0ee05546eeacd42fe09286/llvm/tools/chocolateyinstall.ps1#L8

this link is used by choco. I download this file, not have `llvm-config` binary.

![image](https://user-images.githubusercontent.com/37804932/128870496-87a03623-27cc-42a4-8076-ab0788e463d0.png)

Can you show `llvm-config` path in your system?

```powershell
D:/ $ where llvm-config
D:\opt\llvm\bin\llvm-config.exe
```

@saintfeng

Reply:
related: https://github.com/paritytech/substrate/issues/5682

Reply:
I'm not clear about how llvm works in substrate building. `llvm-config` is not in my path. 
![image](https://user-images.githubusercontent.com/6375317/128955112-90424a24-c99d-4130-aaa9-97cfd6db3f27.png)

Reply:
I think we need to understand why it can compile without llvm-config on your system.